src/open_inwoner/scss/components/Typography/Link.scss

```diff
@@ -83,3 +83,20 @@
     transform: scale(1);
   }
 }
+
+.wysiwyg {
```

Comment:
I think it would be nicer to add a completely new file for all of the 'wysiwyg' styles inside the Typography SCSS directory - because CKEditor probably has many unpleasant side effects,
so in src/open_inwoner/scss/components/Typography perhaps add a 'wysiwyg' file with that as a parent class and then actual tags can be added in the future (we will add Tables at some point too).